app/views/admin_mailer/new_business.text.erb

```diff
@@ -1,3 +1,7 @@
 <%= @business.contact_name %> (<%= @business.user.email %>) added their business!
 
 <%= notification_url(@notification.record) %>
+
+<% if @business.referring_user_email.present? %>
+  Referred by <%= @business.referring_user_email %>
+<% end %>
```

Comment:
This will render two spaces in front of the text on a plain-text email. Let's remove the indentation in these two emails.

Reply:
Done